cmd/kinflate/glossary.md

```diff
@@ -0,0 +1,267 @@
+# Glossary
+
+[DAM]: #declarative-application-management
+[JSON]: https://www.json.org/
+[Resource]: #resource
+[YAML]: http://www.yaml.org/start.html
+[application]: #application
+[apply]: #apply
+[apt]: https://en.wikipedia.org/wiki/APT_(Debian)
+[base]: #base
+[bases]: #base
+[bespoke]: #bespoke-configuration
+[kinflate]: #kinflate
+[manifest]: #manifest
+[off-the-shelf]: #off-the-shelf
+[overlay]: #overlay
+[overlays]: #overlay
+[patch]: #patch
+[patches]: #patch
+[proposal]: https://github.com/kubernetes/community/pull/1629
+[rebase]: https://git-scm.com/docs/git-rebase
+[resource]: #resource
+[resources]: #resource
+[rpm]: https://en.wikipedia.org/wiki/Rpm_(software)
+[target]: #target
+[workflow]: workflows.md
+
+## application
+
+An _application_ is a group of k8s resources that
+server some common purpose, e.g.  a webserver backed by
+a database.
+
+[Resource] labelling, naming and metadata schemes have
+historically served to group resources together for
+collective operations like _list_ and _remove_.
+
+This [proposal] describes a new k8s resource called
+_application_ to more formally describe this idea and
+provide support for application-level operations and
+dashboards.
+
+[kinflate] configures k8s resources, and the proposed
+application resource is just another resource.  There's
+some conceptual overlap between the proposal’s
+application resource and the kinflate [manifest].  The
+application resource has a `Components` field that
+serves a purpose similar to the `resources` field in
+the kinflate manifest.  This overlap can be resolved in
+various ways, the simplest being that kinflate does
+nothing special with the application resource.
+
+
```

Comment:
hmm... not sure if should include "application" here. Do we refer to it anywhere ? Users might think that `kinflate` does something with `application` and the fact it does not.

Reply:
* Brian's doc is still called declarative application management (not declarative configuration mgmt), and we're an impl, so we kinda have to address the "a" word

* we've been asked to describe how kinflate works with the proposed application resource, both by folks here and folks in the app-def WG.  Trying to answer that by saying that at the moment it does nothing special.

Reply:
Sounds good.